.eslintrc

```diff
@@ -25,5 +25,17 @@
   "rules": {
     "valid-jsdoc": [0],
     "require-jsdoc": [0],
+    "quotes": ["error", "backtick"],
+    "jsx-quotes": ["error", "prefer-double"],
+    "max-len": ["error", 80],
+    "semi": ["error", "never"],
+    "object-curly-spacing": ["error", "always"],
+    "comma-dangle": ["error", {
```

Comment:
Yes exactly. The eslint rules become the only source of formatting rules. Which makes perfect sense IMO.